tensorflow/training/docker/2.7/py3/Dockerfile.ec2.cpu.os_scan_allowlist.json

```diff
@@ -0,0 +1,32 @@
+{
+    "tensorflow": [
+        {
+            "description": "[tensorflow](https://pypi.org/project/tensorflow/) is a machine learning framework.\n\nAffected versions of this package are vulnerable to Arbitrary File Write via Archive Extraction (Zip Slip) via a crafted archive when `tf.keras.utils.get_file` is used with `extract=True`.\r\n\r\nNOTE: This CVE is disputed as the vendor's position is that `tf.keras.utils.get_file` is not intended for untrusted archives. However, we feel this advisory is relevant as, as at the time of publication, there is no known security notice or documentation warning users of this behaviour.",
+            "vulnerability_id": "IN1-PYTHON-TENSORFLOW-1315149",
+            "name": "IN1-PYTHON-TENSORFLOW-1315149",
+            "package_name": "tensorflow",
+            "package_details": {
+                "file_path": "usr/local/lib/python3.8/site-packages/tensorflow-2.7.4.dist-info/METADATA",
+                "name": "tensorflow",
+                "package_manager": "PYTHONPKG",
+                "version": "2.7.4",
+                "release": null
+            },
+            "remediation": {
+                "recommendation": {
+                    "text": "None Provided"
+                }
+            },
+            "cvss_v3_score": 9.1,
+            "cvss_v30_score": 0.0,
+            "cvss_v31_score": 9.1,
+            "cvss_v2_score": 0.0,
+            "cvss_v3_severity": "CRITICAL",
+            "source_url": "https://snyk.io/vuln/SNYK-PYTHON-TENSORFLOW-1315149",
+            "source": "SNYK",
+            "severity": "CRITICAL",
+            "status": "ACTIVE",
+            "title": "SNYK-PYTHON-TENSORFLOW-1315149 - tensorflow"
```

Comment:
Can we add `reason_to_ignore` key that would make the entry look something like:

```
{
            "description": "[tensorflow](https://pypi.org/project/tensorflow/) is a machine learning framework.\n\nAffected versions of this package are vulnerable to Arbitrary File Write via Archive Extraction (Zip Slip) via a crafted archive when `tf.keras.utils.get_file` is used with `extract=True`.\r\n\r\nNOTE: This CVE is disputed as the vendor's position is that `tf.keras.utils.get_file` is not intended for untrusted archives. However, we feel this advisory is relevant as, as at the time of publication, there is no known security notice or documentation warning users of this behaviour.",
            "vulnerability_id": "IN1-PYTHON-TENSORFLOW-1315149",
            "name": "IN1-PYTHON-TENSORFLOW-1315149",
            "package_name": "tensorflow",
            "package_details": {
                "file_path": "usr/local/lib/python3.8/site-packages/tensorflow-2.7.4.dist-info/METADATA",
                "name": "tensorflow",
                "package_manager": "PYTHONPKG",
                "version": "2.7.4",
                "release": null
            },
            "remediation": {
                "recommendation": {
                    "text": "None Provided"
                }
            },
            "cvss_v3_score": 9.1,
            "cvss_v30_score": 0.0,
            "cvss_v31_score": 9.1,
            "cvss_v2_score": 0.0,
            "cvss_v3_severity": "CRITICAL",
            "source_url": "https://snyk.io/vuln/SNYK-PYTHON-TENSORFLOW-1315149",
            "source": "SNYK",
            "severity": "CRITICAL",
            "status": "ACTIVE",
            "title": "SNYK-PYTHON-TENSORFLOW-1315149 - tensorflow",
            "reason_to_ignore": "This cannot be fixed due to xyz"
```